dnf-repoconfig-daemon/polkit/org.rpm.dnf.v0.policy

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE policyconfig PUBLIC
+ "-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/PolicyKit/1.0/policyconfig.dtd">
+<policyconfig>
+  <vendor>Dnf</vendor>
+  <vendor_url>http://example.com</vendor_url>
+
+  <!-- Action id must be lower case -->
+  <action id="org.rpm.dnf.v0.rpm.RepoConf.write">
+    <description>Write repository configuration file</description>
+    <message>The message??</message>
```

Comment:
These are human visible strings, and need to be i18n'ed. I. e. you need a .policy.in, add a `gettext-domain=""` attribute, and use the gettext tools (`msgfmt --xml`) to generate a localized file.

Plus of course, adjust this message :)

Of course this doesn't need to block the PR, but it should get on some "release" checklist before this becomes 1.0.